sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,215 @@
+```java
+callClient.deleteCall(callId);
+```
+
+### Start, Stop, Pause, Resume, and Get a recording
+
+Create a ConverationClient: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L27-L36 -->
+```java
+String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
+
+// Your connectionString retrieved from your Azure Communication Service
+String connectionString = "https://<resource-name>.communication.azure.com/;<access-key>";
+
+// Initialize the call client
+final ConversationClientBuilder builder = new ConversationClientBuilder();
+builder.endpoint(endpoint)
+    .connectionString(connectionString);
+ConversationClient conversationClient = builder.buildClient();
+```
+
+Start a Recording: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L48-L51 -->
+```java
+String conversationId = "<conversationId recieved from starting call>";
+String recordingStateCallbackUri = "<webhook endpoint to which calling service can report status>";
+StartCallRecordingResponse response = conversationClient.startRecording(conversationId, recordingStateCallbackUri);
+String recordingId = response.getRecordingId();
+```
+
+Pause a Recording: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L64-L64 -->
+```java
+conversationClient.pauseRecording(conversationId, recordingId);
+```
+
+Resume a Recording: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L76-L76 -->
+```java
+conversationClient.resumeRecording(conversationId, recordingId);
+```
+
+Stop a Recording: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L88-L88 -->
+```java
+conversationClient.stopRecording(conversationId, recordingId);
+```
+
+Get the Recording State: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L101-L109 -->
+```java
+CallRecordingStateResponse callRecordingStateResponse =
+    conversationClient.getRecordingState(conversationId, recordingId);
+
+/**
+ * CallRecordingState: Active, Inactive
+ * If the call has ended, CommunicationErrorException will be thrown. Inactive is
+ * only returned when the recording is paused.
+ */
+CallRecordingState callRecordingState = callRecordingStateResponse.getRecordingState();
+```
+
+### Play Audio Notification in Call
+
+Play Audio: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L121-L105 -->
+```java
+String audioFileUri = "<uri of the file to play>";
+String audioFileId = "<a name to use for caching the audio file>";
+String callbackUri = "<webhook endpoint to which calling service can report status>";
+String context = "<Identifier for correlating responses>";
```

Comment:
Let's keep the capitalization consistent. Identifier is just a noun and does not need to be capitalized if non of the other ones are capitalized.

Reply:
capitalization  removed